copier/main.py

```diff
@@ -417,7 +417,9 @@ def answers_relpath(self) -> Path:
         2. Template default.
         3. Copier default.
         """
-        return self.answers_file or self.template.answers_relpath
+        path = self.answers_file or self.template.answers_relpath
+        template = self.jinja_env.from_string(str(path))
+        return Path(template.render(**self.answers.combined))
```

Comment:
I think you can simplify this part by using [`_render_string()`](https://github.com/copier-org/copier/blob/beb174f64611d1343621ca5b8b5a7d668eee94ab/copier/main.py#L566,L574):

```suggestion
        return Path(self._render_string(str(path)))
```

Reply:
I tried this out but got a recursion error in my tests:
```
tests/test_answersfile_templating.py:42: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
copier/main.py:927: in run_auto
    return run_copy(src_path, dst_path, data, **kwargs)
copier/main.py:890: in run_copy
    with Worker(src_path=src_path, dst_path=Path(dst_path), **kwargs) as worker:
copier/main.py:184: in __exit__
    raise value
copier/main.py:891: in run_copy
    worker.run_copy()
copier/main.py:640: in run_copy
    src_abspath = self.template_copy_root
/usr/local/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/functools.py:1001: in __get__
    val = self.func(instance)
copier/main.py:613: in template_copy_root
    subdir = self._render_string(self.template.subdirectory) or ""
copier/main.py:587: in _render_string
    return tpl.render(**self._render_context())
copier/main.py:242: in _render_context
    "answers_file": self.answers_relpath,
/usr/local/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/functools.py:1001: in __get__
    val = self.func(instance)
copier/main.py:421: in answers_relpath
    return Path(self._render_string(str(path)))
copier/main.py:587: in _render_string
    return tpl.render(**self._render_context())
copier/main.py:242: in _render_context
    "answers_file": self.answers_relpath,
E   RecursionError: maximum recursion depth exceeded while calling a Python object
```

Reply:
Ah, yes, I see. Ignore my suggestion then.